packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -84,6 +84,10 @@ export interface IGCStats {
     updatedNodeCount: number;
     /** The number of data stores whose reference state updated since last GC run. */
     updatedDataStoreCount: number;
+    /** The size of blobs in the summary. */
+    gcTotalBlobSize?: number;
```

Comment:
Are they optional for compat reasons? or because we can't provide this info sometimes?
In the former case, should we (eventually) follow up with a fix to make it required?

Reply:
Moved to IGeneratedSummaryStats, we are not guaranteed to have a GC tree, so we cannot be guaranteed to have GC stats.

Reply:
Also, GC may not be enabled in which case we won't have these fields.